x-pack/plugins/lens/public/id_generator/id_generator.ts

```diff
@@ -8,5 +8,5 @@
 import uuid from 'uuid/v4';
 
 export function generateId() {
-  return uuid();
+  return 'c' + uuid().replaceAll(/-/g, '');
```

Comment:
I added this as a quick-fix to get the feature working - I guess it's fine to keep it in the long run, but we have to make sure there are no issues if an existing Lens vis already uses classical uuid ids and the user transitions to formula